elizabeth/core/interdata/code.py

```diff
@@ -50,7 +50,8 @@
     "pt": "972",
     "pt-br": "85",
     "ru": "5",
-    "sv": "91"
+    "sv": "91",
+    "tr": "90"
```

Comment:
According to [this page](https://en.wikipedia.org/wiki/List_of_ISBN_identifier_groups) the Turkish ISBN group is 605